doc/content/introduction/installation/_index.md

```diff
@@ -65,22 +73,22 @@ Add user to the docker group:
   More information in:
   https://linoxide.com/linux-how-to/use-docker-without-sudo-ubuntu/
 
-Install the `protoc` Protobuf compiler:[^gopath_deprecation]
-
-	> protoc --version
-	...
-
 
 ## Checkout sources
 
   > git clone https://github.com/QMSTR/qmstr.git
+  
 
 ## Installing the clients
 
 The main entry point into the installation tasks for Quartermaster is
 the Makefile in the main repository. The default installation prefix is `/usr/local`.
 The client programs will be installed into the `bin/` subdirectory:
   
+  > cd qmstr/
+
+  > go install github.com/golang/protobuf/protoc-gen-go
+
```

Comment:
This should be included in the section ##Prerequisites, it's not related here.

Reply:
To install protoc-gen-go you need to be in the same directory as go.mod thus inside qmstr.